src/Configuration/SamlConfiguration.php

```diff
@@ -50,6 +50,21 @@ public function isRolesResetOnLogin(): bool
         return $this->configuration->isSamlRolesResetOnLogin();
     }
 
+    public function getTeamsAttribute(): ?string
+    {
+        return $this->configuration->getSamlTeamsAttribute();
+    }
+
+    public function getTeamsMapping(): array
```

Comment:
```suggestion
    /**
     * @return array<int, array<'saml'|'kimai'|'leader', string>>
     */
     public function getTeamsMapping(): array
```

Add the same for the Interface. Then you can remove the PHPStan ignores as well